tests/app-definitions/TestApplicationString-output.json

```diff
@@ -44,5 +40,9 @@
   "instances": 2,
   "mem": 64,
   "ports": null,
-  "dependencies": null
+  "dependencies": null,
+  "env": {
```

Comment:
Any particular reason you moved this around? If no particular reason, then I'd prefer to have it sit at its old location in order to minimize the PR diff.